package.json

```diff
@@ -8,6 +8,7 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "lint": "eslint src --ext '[.ts, .tsx]' --resolve-plugins-relative-to .",
```

Comment:
Change to this so that the linting will be also run as part of CI

```json
"lint": "yarn run lint:ts && yarn run lint:styles",
"lint:ts": "eslint src --ext '[.ts, .tsx]' --resolve-plugins-relative-to .",
"lint:styles": "stylelint \"src/**/*.css\"",
```